package.json

```diff
@@ -29,7 +29,8 @@
   ],
   "scripts": {
     "prepare": "husky install",
-    "setup": "yarn && yarn build",
+    "setup": "yarn && yarn clean && yarn build",
```

Comment:
I'm not 100% certain on whether we should do `yarn && yarn clean` or `yarn clean && yarn`. Happy to hear your suggestions :slightly_smiling_face:

Reply:
I think clean should come first, because in theory the old files we're cleaning out could include a package.json that is getting its modules installed unnecessarily.

Also, there used to be an actual "yarn clean" command a while back, we should make sure it's completely removed in all versions of yarn we support or this could potentially cause problems.

Reply:
I took a look and from what I gathered, `clean` has been renamed to `autoclean` and is not part of either yarn 1 or 2.
I think we can safely proceed with that naming.

As for the order I think you're right, I'll put yarn clean before yarn :slightly_smiling_face:

Reply:
Ping @alexandrebodin just in case there are some things I'm not aware of regarding the yarn setup?